app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCTableView.scala

```diff
@@ -6,16 +6,21 @@ import scalafx.beans.property.StringProperty
 import scalafx.geometry.Insets
 import scalafx.scene.control.{ContextMenu, MenuItem, TableColumn, TableView}
 
+import java.awt.Toolkit.getDefaultToolkit
+import java.awt.datatransfer.StringSelection
+
 class DLCTableView(model: DLCPaneModel) {
 
   val tableView: TableView[DLCStatus] = {
-    val paramHashCol = new TableColumn[DLCStatus, String] {
-      text = "Temp Contract Id"
+
+    val eventIdCol = new TableColumn[DLCStatus, String] {
+      text = "Event Id"
       prefWidth = 150
       cellValueFactory = { status =>
-        new StringProperty(status,
-                           "Temp Contract Id",
-                           status.value.tempContractId.hex)
+        val eventIdStr =
+          status.value.oracleInfo.singleOracleInfos.head.announcement.eventTLV.eventId
```

Comment:
is this always safe? I see `.head` which can throw no such element

Reply:
Yeah, we have to have at least one oracle